transformations/chinese_butter_fingers_perturbation/transformation.py

```diff
@@ -0,0 +1,248 @@
+import itertools
+import random
+import unidecode as u
+import json
+import os
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import jieba
+import pypinyin
+
+
+def chinese_butter_finger(text,
+                          chinese_character_database,
+                          common_chinese_character_database,
+                          chinese_words_database,
+                          prob,
+                          rare_word_prob,
+                          seed,
+                          max_outputs,
+                          consider_tone,
+                          word_level_perturb):
+    random.seed(seed)
+    perturbed_texts = []
+    for _ in range(max_outputs):
+        butter_text = ""
+        output = jieba.lcut(text)
+        if(word_level_perturb):
+            words_and_similar_word_dict_list = get_words_with_similar_pinyin(output,
+                                                                            rare_word_prob,
+                                                                            chinese_character_database,
+                                                                            common_chinese_character_database,
+                                                                            chinese_words_database,
+                                                                            consider_tone)
+            for dict in words_and_similar_word_dict_list:
+                perturb_word = dict['perturb_word']
```

Comment:
Since you've changed the `perturb_word` to `original_word` in the `get_similar_word_pinyin_list ` function, could you also change the variable names here to keep consistent?

Reply:
Done!